composer.json

```diff
@@ -29,6 +29,7 @@
         "sonata-project/core-bundle": "~2.3,>=2.3.1"
     },
     "require-dev": {
+        "rhumsaa/uuid": "^2.8",
```

Comment:
I could do that, but i'm also checking if the [correct type is returned](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/523/files#diff-304847c39531513d83f2682805eb7b6cR321)

Or should I mock the whole uuid class and doctrine mapping etc?